module/documents/active-effect.mjs

```diff
@@ -618,6 +631,34 @@ export default class ActiveEffect5e extends ActiveEffect {
 
   /* -------------------------------------------- */
 
+  /**
+   * Add modifications to the core ActiveEffect config.
+   * @param {ActiveEffectConfig} app   The ActiveEffect config.
+   * @param {jQuery|HTMLElement} html  The ActiveEffect config element.
+   */
+  static onRenderActiveEffectConfig(app, html) {
+    // TODO: rebase once #4805 is merged.
+    if ( game.release.generation < 13 ) html = html[0];
+    const element = new foundry.data.fields.SetField(new foundry.data.fields.StringField(), {}).toFormGroup({
+      label: game.i18n.localize("DND5E.CONDITIONS.RiderConditions.label"),
+      hint: game.i18n.localize("DND5E.CONDITIONS.RiderConditions.hint")
+    }, {
+      name: "flags.dnd5e.riders.statuses",
+      value: app.document.getFlag("dnd5e", "riders.statuses") ?? [],
+      options: CONFIG.statusEffects.map(se => ({ value: se.id, label: se.name }))
+    }).outerHTML;
+    html.querySelector("[data-tab=details] > .form-group:last-child").insertAdjacentHTML("afterend", element);
+
+    if ( game.release.generation < 13 ) {
+      html.querySelector(".form-fields:has([name=statuses])").insertAdjacentHTML("afterend", `
```

Comment:
In v13 this was being appended twice because the localisation hint was already being applied by the core application. I guess that's not working in v12? Worth checking if appending this is even necessary.

Reply:
There is no `hint` in v12 if that is what you mean?

Reply:
I mean the localisation prefix wasn't applying the hint in v12. Maybe it's a v13-only feature. I've been on v13 so long I don't remember.

Reply:
In v12, it doesn't use `{{formGroup}}`.
```html
        <div class="form-group">
            <label>Status Conditions</label>
            <div class="form-fields">
                <multi-select name="statuses">
                    {{#each statuses as |status|}}
                    <option value="{{status.id}}" {{status.selected}}>{{status.label}}</option>
                    {{/each}}
                </multi-select>
            </div>
        </div>
```